src/MatrixFields/band_matrix_row.jl

```diff
@@ -135,3 +135,9 @@ Base.:*(value::SingleValue, row::BandMatrixRow) =
 
 Base.:/(row::BandMatrixRow, value::Number) =
     map(entry -> rdiv(entry, value), row)
+
+inv(row::DiagonalMatrixRow) = DiagonalMatrixRow(inv(row[0]))
```

Comment:
```suggestion
inv(row::DiagonalMatrixRow) = DiagonalMatrixRow(inv(row[1]))
```
is there a custom `getindex` for `DiagonalMatrixRow`?

Reply:
Yes, it indexes using the diagonal index. So -1 corresponds to the first lower diagonal, 1 corresponds to the first upper diagonal, etc.